### ISSUE TITLE
Count events by "First time ever for user" in Trends

### ISSUE THREAD

**Issue by Twixes** [PostHog/posthog#15945]
## Is your feature request related to a problem?

It would be useful to count how many events happened for the first time ever (per user).

## Describe the solution you'd like

In addition to "Total count" or "Unique users" there should be a "First time for user" option.

## Additional context

This was requested by a user: ZEN-3437. Related, but focused on funnels and session recordings: #11941

**Comment by ConstantinSchreiber** [comment]
+1

Would love to see this feature being added since the Plugin that solved the issue before just has been disabled for all new projects.

**Comment by MarconLP** [comment]
There are two methods to achieve this:
1. Creating a cohort using the `completed an event for the first time` filter, check [this page](https://posthog.com/tutorials/track-new-returning-users#completed-event-for-the-first-time-cohort) for instructions.
2. Manually setting a user property, check [this page](https://posthog.com/tutorials/track-new-returning-users#manually-set-user-property) for instructions.

**Comment by subproject22** [comment]
> There are two methods to achieve this:
> 
> 1. Creating a cohort using the `completed an event for the first time` filter, check [this page](https://posthog.com/tutorials/track-new-returning-users#completed-event-for-the-first-time-cohort) for instructions.
> 2. Manually setting a user property, check [this page](https://posthog.com/tutorials/track-new-returning-users#manually-set-user-property) for instructions.

No such event exist in my dashboard

<img width="531" alt="image" src="https://github.com/PostHog/posthog/assets/129603523/68a3fe11-e5d1-4149-89aa-1c34b94b13c6">

How do I track first time users then? Where's `completed an event for the first time`?

**Comment by MarconLP** [comment]
Hey @subproject22, that option is only available when subscribed to the `pay-as-you-go` tier. Check out [the billing page](https://posthog.com/pricing) for more details.

**Comment by subproject22** [comment]
@MarconLP thanks for response, but I don't see anything close to this on the page you linked.

Do I need to sign up to "Ridiculously cheap" plan and this option will appear? I had to say pricing plans are very confusing.

**Comment by MarconLP** [comment]
@subproject22 Yes - you'll need to upgrade to the `Ridiculously cheap` plan to view that option.

**Comment by Twixes** [comment]
This is now available!
